recipes/aiomysql/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "aiomysql" %}
+{% set version = "0.0.9" %}
+{% set sha256 = "3457baca19a1c307ffbab8e927259fcbf3144a5852ab4cf4205dc26e7c05623f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<33]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - asyncio  # [py<34]
+    - pymysql >=0.7.5
+
+test:
+  imports:
+    - aiomysql
+
+about:
+  home: https://github.com/aio-libs/aiomysql
+  summary: "aiomysql is a library for accessing a MySQL database from the asyncio"
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jettify
```

Comment:
@jettify, are you interested in [being a co-maintainer](http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role) for the conda-forge package of `aiomysql`?

Reply:
Thanks for your offer, unfortunately I am have lots of projects to maintain at https://github.com/aio-libs and doubt I will manage also contribute here.